### PR TITLE
Add Plot Wizard inside "Visualizing Plots" User Guide

### LINKED ISSUE #4885: VS Code plots wizard
### Report

Add plots wizard as a way to define plots in https://dvc.org/doc/user-guide/experiment-management/visualizing-plots

### PR DESCRIPTION
**Pull request by julieg18**
* updates `/doc/user-guide/experiment-management/visualizing-plots#defining-plots` with a section for the VSCode Extension's Plot Wizard

## Demo

image

Fixes #4885

### CODE REVIEW

File: content/docs/user-guide/experiment-management/visualizing-plots.md [hunk 1]
```diff
@@ -172,6 +172,14 @@ file:///Users/usr/src/dvc_plots/index.html
 
 [template]: #plot-templates-data-series-only
 
+### VSCode Extension Plots Wizard
+
+Use the
+[DVC Extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+to define plots inside your `dvc.yaml`.
+
+https://youtu.be/qG2gfTuQWtk?si=ffpkBLX-odowXK-G
```
Comment (julieg18) [inline content/docs/user-guide/experiment-management/visualizing-plots.md:181]: I used the update video but we could use a GIF instead 🤔

**Comment by dberenbaum** [comment]
Thanks @julieg18! Let's move it to the top of `Defining Plots` to make it more prominent and save people time from having to read all the other info only to find out at the end they could just use the plots wizard. I think we can drop the `VSCode Extension Plots Wizard` header if we do that.

**Comment by julieg18** [comment]
> Thanks @julieg18! Let's move it to the top of Defining Plots to make it more prominent and save people time from having to read all the other info only to find out at the end they could just use the plots wizard. I think we can drop the VSCode Extension Plots Wizard header if we do that.

Makes sense, moved to the top!

**Comment by dberenbaum** [comment]
Sorry @julieg18, I meant let's move it to the top of the `Defining Plots` section, not the whole page.

**Comment by julieg18** [comment]
> Sorry @julieg18, I meant let's move it to the top of the Defining Plots section, not the whole page.

Whoops, apologies, I misread `Defining Plots` as `Visualizing Plots` 😅.  Moved to the top of `Defining Plots`!

**Review by dberenbaum — APPROVED** [review]

**Comment by dberenbaum** [comment]
Thanks @julieg18! After reading it, I moved a bit of the intro text above it to give some more context.

**Pull request merged**